### ISSUE TITLE
Training model on KITTI

### ISSUE THREAD

**Issue by xueyaqing** [xingyizhou/CenterTrack#142]
I use experiments/kitti_half.sh to train kitti_half model,but I only got 85.86 MOTA, the arguements I used are totally the same as yours,I think the jitter is too large,is there something else should I notice when I train the model?

**Comment by xingyizhou** [comment]
Thank you for letting me know this. I do have observed considerable randomness on the KITTI model. The 5 repeated experiments I run before release give 88.7, 87.5, 88.1, 87.7, 85.9 MOTA, respectively. If that happens constantly, I would recommend you to use the exact pytorch version (1.0) I used in the release. Also, one small thing that might matter is to disable cudnn batch norm (Open ~/anaconda3/envs/CenterTrack/lib/python3.6/site-packages/torch/nn/functional.py and find the line with torch.batch_norm and replace the torch.backends.cudnn.enabled with False). This is a known issue (see related links in CenterNet installation) for torch < 1.0, but should have been fixed for torch >= 1.0. I did this for all my torch environment anyway.

**Comment by yustaub** [comment]
Can you reproduce the 88.7 MOTA now? I can only get 86.5 MOTA. @xueyaqing

**Comment by zhangchuang-zc** [comment]
> Can you reproduce the 88.7 MOTA now? I can only get 86.5 MOTA. @xueyaqing

Have you solved this problem yet? I can only get 86.59 MOTA. @yustaub
